Show an org's events in reverse chronological order.

timparenti opened the pull request:
Base this on the representative dates of the events, not just the (implicit) order the events were added to the database.

timparenti commented:
Hm… since the page presents a history of all an org's events, I'd been thinking more by analogy to the main [listing of past events](https://abtech.andrew.cmu.edu/tracker/events/past), but of course the difference here is that an event need not yet be in the past in order to be listed on the org pages.

@auy86 @bridget-tan Thoughts on the direction of the sort here?  Organizations with a lot of events currently [require scrolling](https://abtech.andrew.cmu.edu/tracker/organizations/5) to get to the events that are most current.

A potential, but more complex, compromise could involve separating out and listing an org's current/future events (chronologically) above their past events (reverse chronologically).

DaAwesomeP commented:
I think the ascending makes sense here since you normally use this page to lookup previous events of an organization. The compromise option does make sense but I don't think we run into this issue that often. Will reopen if more changes pushed.

timparenti commented:
Rebased, updated, and pushed based on feedback from @DaAwesomeP that ascending order is in fact desired, so as to fix the actual bug here (events appearing in database order instead of any logical order), aesthetic considerations aside.  The commit doesn't seem to have made it here, though?

timparenti commented:
Since it looks like this will need a new PR unless you're willing to merge the branch by hand… @DaAwesomeP, is the compromise proposal from 2019-09-27 actually _desirable_ here? Or is merely fixing the extant bug in the ordering sufficient?
